.storybook/webpack.config.js

```diff
@@ -33,6 +33,5 @@ module.exports = ({ config }) => {
   });
   config.resolve.modules.push('app');
   config.resolve.extensions.push('.js', '.jsx', '.react.js');
-  config.module.rules[0].use[0].options.plugins = [require.resolve('babel-plugin-react-docgen')];
```

Comment:
This is removed because the doc generation is done by default on the new version ?

Reply:
Yes , Post v6 doc-gen support is by default.